### PR TITLE
Added bootstrapping script for dev container to update Rust and Wasm3…

### PR DESCRIPTION
**Pull request by suneetnangia**
This allows dev container to install a latest stable version of Rust toolchain and the required wasm32 targets i.e. wasm32-wasi wasm32-unknown-unknown

### CODE REVIEW

**Review by itowlson — APPROVED** [review]
Looks good to me - thanks!

**Comment by itowlson** [comment]
@suneetnangia it looks like you need to GPG-sign the commit - the link in the "merging is blocked" message has the info.  Apologies for not spotting this earlier!

**Comment by suneetnangia** [comment]
> @suneetnangia it looks like you need to GPG-sign the commit - the link in the "merging is blocked" message has the info. Apologies for not spotting this earlier!

Thanks, done now, see if that's ok.

File: examples/spin-timer/Cargo.lock [hunk 1]
```diff
@@ -3481,6 +3481,18 @@ dependencies = [
  "wit-bindgen-wasmtime",
 ]
 
+[[package]]
```
Comment (rylev) [inline examples/spin-timer/Cargo.lock:3484]: I'm confused. Why are these changes part of this PR? Perhaps these are unrelated?

**Review by rylev — CHANGES_REQUESTED** [review]

File: examples/spin-timer/Cargo.lock (hunk 1 above)
Comment (suneetnangia) [reply examples/spin-timer/Cargo.lock:3484]: That's by error, let me remove. Thanks for spotting it.

Comment (suneetnangia) [reply examples/spin-timer/Cargo.lock]: Done.

File: Cargo.lock [hunk 2]
```diff
@@ -2129,9 +2129,9 @@ dependencies = [
 
 [[package]]
 name = "h2"
-version = "0.3.16"
+version = "0.3.17"
```
Comment (rylev) [inline Cargo.lock:2132]: I think this is the same issue as the other file. This lock file should not need to be updated.

Comment (suneetnangia) [reply Cargo.lock]: Done...

**Review by rylev — APPROVED** [review]
🎉 Thank you!
